mxcube3/routes/SampleCentring.py

```diff
@@ -264,8 +294,16 @@ def update_shapes():
         # Shape does not have any refs, create a new Centered position
         if not refs:
             x, y = shape_data["screen_coord"]
-            mpos = mxcube.diffractometer.get_centred_point_from_coord(x, y)
-            shape = mxcube.shapes.add_shape_from_mpos([mpos], (x, y), t)
+            mpos = mxcube.diffractometer.get_centred_point_from_coord(x, y, return_by_names=True)
+            pos.append(mpos)
+            # MD3 needs the center of the grid instead of the top left corner
```

Comment:
The position of the Shape object should always be top left corner, we can add a method or similar that calculates the center position if needed. This could also and should perhaps be handled in the diffractometer specific code. I guess it wont work for routines that expect position to be top left corner like it is now ?